src/integration-tests/integration-test.py

```diff
@@ -106,12 +107,21 @@ class Integration(BaseModel):
             ContainerType.unique_reports: 1,
             ContainerType.coverage: 1,
             ContainerType.inputs: 2,
+            # TODO: crashdumps are intermittently not captured
```

Comment:
I've thought about this as well, but I think that writing the dump is synchronous: the ASAN runtime does it before the process exits. Dirwatcher is a possible solution but it would be hard to match up dumps and crashes, at the moment we make the assumption that if we see a dump then it matches the (single) crash that was just found, so we can rename them to match. If we're finding the dumps later on we don't know what crash to associate them with.

Reply:
Ah that's a good point, if there were a way for us to control where the dump gets dropped, that could be an option. If we knew if it's placed in the working directory, or always adjacent to the location of the binary, we could create a folder per libfuzzer invocation (and a DirWatcher per folder) so we know which dump is associated with which crash.

Reply:
I think I'll open a separate workitem for this. I think running the fuzzer in a temporary folder each time is probably a good idea even in general, since it means that we start with a fresh slate each time.